src/Framework/test/SharedFxTests.cs

```diff
@@ -302,14 +302,12 @@ public void RuntimeListListsContainsCorrectEntries()
         [Fact]
         public void RuntimeListListsContainsCorrectPaths()
         {
-            var runtimePath = Environment.GetEnvironmentVariable("ASPNET_RUNTIME_PATH");
-            if (string.IsNullOrEmpty(runtimePath))
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("ASPNET_RUNTIME_PATH")))
             {
                 return;
             }
 
```

Comment:
Unless this is for the odd case of running these tests in a local build i.e. not on a Helix agent, it doesn't seem useful anymore.
```suggestion
```
If it is for that odd case, add a comment.

Reply:
Sounds good, I'll remove all these checks